crypto/fipsmodule/sha/sha3.c

```diff
@@ -377,3 +377,48 @@ int SHAKE_Squeeze(uint8_t *md, KECCAK1600_CTX *ctx, size_t len) {
   //FIPS_service_indicator_update_state();
   return 1;
 }
+
+int SHAKE128_Init_x4(KECCAK1600_CTX_x4 *ctx) {
```

Comment:
Can this function actually fail? If so, when?

Reply:
I added a comment https://github.com/aws/aws-lc/blob/4fca7ae7122f5f6d0c15625f90a3211e54f76772/crypto/fipsmodule/sha/internal.h#L464.

Reply:
If the function always succeeds, can we drop the return code?

Reply:
I think it is better to keep it. It is, indeed, a function designed specifically for mlkem, nevertheless, nothing prevents from calling it in different use case scenario (if some). Wouldn't it be better to document the _skipped_ return code check in mlkem-native, and argument it with the input values?

Reply:
> I think it is better to keep it. It is, indeed, a function designed specifically for mlkem, nevertheless, nothing prevents from calling it in different use case scenario (if some). 

Yes, but my understanding is that this function will never fail, regardless of who calls it?